scheduler/src/cook/mesos/api.clj

```diff
@@ -741,7 +741,7 @@
                  :max_retries (:job/max-retries job) ; consistent with input
                  :retries_remaining (- (:job/max-retries job) (util/job-ent->attempts-consumed db job))
                  :max_runtime (:job/max-runtime job Long/MAX_VALUE) ; consistent with input
-                 :framework_id fid
+                 :framework_id (str fid)
```

Comment:
Actually, I don't think `str` is the right thing. Let me check this first before we merge.